key-vault-node-quickstart-v4/index.js

```diff
@@ -0,0 +1,39 @@
+var http = require('http');
+const {DefaultAzureCredential} = require('@azure/identity');
+const {SecretsClient} = require('@azure/keyvault-secrets');
+
+var server = http.createServer(function(request, response) {
+    response.writeHead(200, {"Content-Type": "text/plain"});
+});
+
+async function main(){
+// DefaultAzureCredential expects the following three environment variables:
+// - AZURE_TENANT_ID: The tenant ID in Azure Active Directory
+// - AZURE_CLIENT_ID: The application (client) ID registered in the AAD tenant
+// - AZURE_CLIENT_SECRET: The client secret for the registered application
+const credential = new DefaultAzureCredential();
+  
+const vaultName = process.env["KEYVAULT_NAME"] || "<YourVaultName>";
+const url = `https://${vaultName}.vault.azure.net`;
+  
+const client = new SecretsClient(url, credential);
+  
+  // Create a secret
+  const secretName = "MySecretName";
```

Comment:
Please have all const at the top of the file so the user doesn't have to search the sample for them.